userspace/libscap/engine/source_plugin/source_plugin.h

```diff
@@ -18,6 +18,8 @@ limitations under the License.
 
 #include <stdint.h>
 #include "plugin_info.h"
+#include "scap.h"
```

Comment:
Okay, we can't use a forward declaration here, but please find the `scap_stats_v2` struct a better home than scap.h. `scap_stats.h`? A new dedicated header? We can include it from scap.h but let's try to minimize the includes of the public API in implementation files.